plugins/zoom/help.md

```diff
@@ -280,7 +280,7 @@ Example output:
 |Host Key|string|None|False|User's host key|123321|
 |ID|string|None|True|User identifier|T9ti3NBxR42swGKrqABGig|
 |IM Group IDs|[]string|None|False|IM IDs of the groups user belongs to|['t-_-d56CSWG-7BF15LLrOw', 't-_-d56CSWG-7BF15LLrow']|
-|JID|string|None|False|JID of user|T9ti3NBxR42swGKrqABGig@xmpp.zoom.us|
+|JID|string|None|False|JID of user|user@example.com|
```

Comment:
Oh I see, because it contains an `@` it considers it a failing email address. Keep it how it is for now and I can make a ticket for the validators to change it to a warning rather than break 😄